packages/react-components/src/button/src/Button.css

```diff
@@ -33,25 +35,71 @@
     width: max-content;
 }
 
+/* STATES | FOCUS */
+.o-ui-button:focus,
+.o-ui-button-focus {
+    outline: none;
+    z-index: 2;
+}
+
+/* STATES | FOCUS | FOCUS RING */
+.o-ui-button:after {
+    opacity: 0;
+    content: "";
+    top: -2px;
+    left: -2px;
+    right: -2px;
+    bottom: -2px;
```

Comment:
How come our focus ring is now that complex? Used to be a simple box-shadow.

These focus ring position values should be extracted to vars in `index.css`

Reply:
Since box-shadow does not supports an offset, on any background other than white you would see a 2px solid white border.